include/swift/Strings.h

```diff
@@ -39,9 +39,9 @@ namespace swift {
   static const char LLDB_EXPRESSIONS_MODULE_NAME_PREFIX[] = "__lldb_expr_";
 
   /// The name of the fake module used to hold imported Objective-C things.
-  static const char MANGLING_MODULE_OBJC[] = "__ObjC";
-  /// The name of the fake module used to hold imported C things.
-  static const char MANGLING_MODULE_C[] = "__C";
+  static const char MANGLING_MODULE_OBJC[] = "__C";
```

Comment:
I didn't want to swap the meaning of the two symbol names up front because I wanted to make sure I had found all of the places that needed to change. I can do that in a follow-up commit.